cmd/crio/config.go

```diff
@@ -107,7 +115,20 @@ runtime_untrusted_workload = "{{ .RuntimeUntrustedWorkload }}"
 #     privileged containers are by definition trusted and will always use the
 #     trusted container runtime. If default_container_trust is set to "trusted",
 #     CRI-O will use the trusted container runtime for all containers.
-default_workload_trust = "{{ .DefaultWorkloadTrust }}"
+#
+# DEPRECATED: The runtime handler should provide a key to the map of runtimes,
+#     avoiding the need to rely on the level of trust of the workload to choose
+#     an appropriate runtime.
+#     The support of this option will continue through versions 1.12 and 1.13.
+#     By version 1.14, this option will no longer exist.
+#default_workload_trust = "{{ .DefaultWorkloadTrust }}"
+
+  # The "crio.runtime.runtimes" table defines a list of OCI compatible runtimes.
+  # The runtime to use is picked based on the runtime_handler provided by the CRI.
+  # If no runtime_handler is provided, the runtime will be picked based on the level
+  # of trust of the workload.
+  [crio.runtime.runtimes.runtime_handler_name]
```

Comment:
this is killing the configuration parsing in the config subcommand. Basically, every option below this now aren't getting picked up anymore from the config. I've fixed it in the very latest commit